tensorflow_examples/models/custom_resnet/load_data.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Download the TinyImageNet dataset and get the ImageDataGenerator objects.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pandas as pd
+import numpy as np
+import occlusion
+import tensorflow
+assert tf.__version__.startswith('2')
+
+from tensorflow.keras.preprocessing.image import ImageDataGenerator
+
+VAL_ANNOT = "/tiny-imagenet-200/val/val_annotations.txt"
+TRAIN = "/content/tiny-imagenet-200/train/"
+VAL = "/content/tiny-imagenet-200/val/images"
+
+data_url = "http://cs231n.stanford.edu/tiny-imagenet-200.zip"
+
+def occlusion(thresh_prob=0.5):
```

Comment:
This is never used.

Reply:
I have removed this as the overhead outweighed the regularization benefits cutout offered. The generator had to get the images processed by the external function and proved to be a bottleneck